### ISSUE TITLE
WARNING: The scripts populate_json, prancer, register_key_in_azure_vault, terraform_to_json and validator are installed in '/home/r4redu/.local/bin' which is not on PATH.

### ISSUE THREAD

**Issue by rezoan** [prancer-io/cloud-validation-framework#288]
In my `ubuntu 20.04.2.0`, i have `python 2.7.18` and `pip3 20.0.2`.
I was trying to install prancer-basic via `pip3 install prancer-basic`
It get installed successfully with below warning:
```
WARNING: The scripts populate_json, prancer, register_key_in_azure_vault, terraform_to_json and validator are installed in '/home/r4redu/.local/bin' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
Successfully installed prancer-basic-1.1.17
```
Now when i try to see the version via `prancer --version` it is saying `prancer: command not found.`

**Do i need to do some manual work here to get it working?**

**Comment by jaiminswan** [comment]
We will update setup.py to include it in bin directory
